docs/source/sections/Installation.rst

```diff
@@ -6,27 +6,29 @@ In this section we will outline the installation steps for installing the softwa
 
 The first stage is installing a version of Python 3.9, if you don't have a working version. We suggest installing a working Anaconda distribution from https://www.anaconda.com/products/distribution#macos following the instructions on that page.
```

Comment:
Should this be edited to remove reference to anaconda/python 3.9? As here: https://github.com/antarctica/PolarRoute/blob/docs_update/docs/source/sections/Installation.rst?plain=1